replicatedhq/ship
# [WIP] integration tests

What I Did
------------
1. Added a set of integration tests that run ship with a saved state file (and provided customer id/installation id/release semver) and then compare the output with a stored copy

2. Added a CLI flag `state-file` that defines the path to the state file that ship should read from (not write to)

3. Made the CLI flag `release-semver` determine what release version to run if provided

How I Did it
------------


How to verify it
------------
Run the integration tests with `make integration-test` or wait for it to run on CircleCI.

Description for the Changelog
------------

Added a CLI flag `state-file` that defines the path to the state file that ship should read from (not write to)

Made the CLI flag `release-semver` determine what release version to run if provided


Picture of a ~Boat~Ship (not required but encouraged)
------------


![160421-n-ye579-005-1](https://user-images.githubusercontent.com/2318911/41382097-7a44569a-6f1f-11e8-8e72-44374ead4e07.jpg)
